content: add summit recap blog

olehpratsko opened the pull request.

pditommaso commented on content/blog/2022/nextflow-summit-2022-recap.md:
```diff
@@ -0,0 +1,92 @@
+
+The three-day Nextflow Summit featured 33 talks delivered by speakers from academia, research, healthcare providers, biotechs, and cloud providers. This year’s speakers came from the following organizations:
+
+- University of Maryland
+- Wyoming Public Health Laboratory
+- Wellcome Sanger Institute
+- Quantitative Biology Center, University of Tübingen
+- Curative
+- Center for Genomic Regulation
+- Janelia Research Campus
+- Seqera Labs
+- Amazon Web Services
+- Google
+- Quilt Data
+- UNC Lineberger Comprehensive Cancer Center
+- Università degli Studi di Macerata
+- Enterome
+- Quadram Institute BioScience
+- Oxford Nanopore
+- Chan Zukerberg Biohub
+- Microsoft
+- DNA Nexus
+- Centre for Molecular Medicine and Therapeutics, University of British Columbia
```
Worth to sort alphabetically

pditommaso commented on content/blog/2022/nextflow-summit-2022-recap.md:
```diff
@@ -0,0 +1,92 @@
+Several other announcements were made during [Evan Floden’s talk](https://www.youtube.com/watch?v=yJpN3fRSClA&list=PLPZ8WHdZGxmUdAJlHowo7zL2pN3x97d32&index=22&t=127s), such as:
+
+- MultiQC is joining the Seqera Labs family
+- Fusionfs – a distributed virtual file system for cloud-native data pipelines
```
```suggestion
- Fusion – a distributed virtual file system for cloud-native data pipelines
```

pditommaso commented:
The image file `assets/img/nextflow-summit-2022-recap.png` is 9 MB, definitely too big for a web page.

olehpratsko commented:
> The image file `assets/img/nextflow-summit-2022-recap.png` is 9 MB, definitely too big for a web page.

I've changed format from png to jpg and adjusted size of the picture.

abhi18av commented on content/blog/2022/nextflow-summit-2022-recap.md:
```diff
@@ -0,0 +1,92 @@
+
+The three-day Nextflow Summit featured 33 talks delivered by speakers from academia, research, healthcare providers, biotechs, and cloud providers. This year’s speakers came from the following organizations:
+
+- Amazon Web Services
+- Center for Genomic Regulation
+- Centre for Molecular Medicine and Therapeutics, University of British Columbia
+- Chan Zukerberg Biohub
+- Curative
+- DNA Nexus
```
Maybe we should add links to their websites?

abhi18av commented on content/blog/2022/nextflow-summit-2022-recap.md:
```diff
@@ -0,0 +1,92 @@
+
+The three-day Nextflow Summit featured 33 talks delivered by speakers from academia, research, healthcare providers, biotechs, and cloud providers. This year’s speakers came from the following organizations:
+
+- Amazon Web Services
+- Center for Genomic Regulation
+- Centre for Molecular Medicine and Therapeutics, University of British Columbia
+- Chan Zukerberg Biohub
+- Curative
+- DNA Nexus
+- Enterome
+- Google
+- Janelia Research Campus
+- Microsoft
+- Oxford Nanopore
+- Quadram Institute BioScience
+- Seqera Labs
+- Quantitative Biology Center, University of Tübingen
+- Quilt Data
+- UNC Lineberger Comprehensive Cancer Center
+- Università degli Studi di Macerata
+- University of Maryland
+- Wellcome Sanger Institute
+- Wyoming Public Health Laboratory
```
Maybe we should add links to the corporate/research-group websites?

abhi18av commented on content/blog/2022/nextflow-summit-2022-recap.md:
```diff
@@ -0,0 +1,92 @@
+
+## Data provenance tracking
+
+In the Thursday morning keynote, Rob Patro﹘Associate Professor at the University of Maryland Dept. of Computer Science and  CTO and co-founder of Ocean Genomics﹘described in his talk “[What could be next(flow)](https://www.youtube.com/watch?v=vNrKFT5eT8U&list=PLPZ8WHdZGxmUdAJlHowo7zL2pN3x97d32&index=6),” how far the Nextflow community had come in solving problems such as reproducibility, scale, m ...
```
```suggestion
In the Thursday morning keynote, Rob Patro﹘Associate Professor at the University of Maryland Dept. of Computer Science and CTO and co-founder of Ocean Genomics﹘described in his talk “[What could be next(flow)](https://www.youtube.com/watch?v=vNrKFT5eT8U&list=PLPZ8WHdZGxmUdAJlHowo7zL2pN3x97d32&index=6),” how far the Nextflow community had come in solving problems such as reproducibility, scale, modularity, and ease of use. He then challenged the community with some complex issues still waiting in the wings. He focused on data provenance as a particularly vexing challenge explaining how tremendous effort currently goes into manual metadata curation.
```

abhi18av commented on content/blog/2022/nextflow-summit-2022-recap.md:
```diff
@@ -0,0 +1,92 @@
+
+Rob offered suggestions about how Nextflow might evolve, and coined the term “augmented execution contexts” (AECs) drawing from his work on provenance tracking – answering questions such as “what are these files, and where did they come from.” This thinking is reflected in [tximeta](https://github.com/mikelove/tximeta), a  project co-developed with Mike Love of UNC. Rob also proposed ideas around ...
+
+In his talk with the clever subtitle “[one link to rule them all](https://www.youtube.com/watch?v=dttkcuP3OBc&list=PLPZ8WHdZGxmUdAJlHowo7zL2pN3x97d32&index=13),” Aneesh Karve of Quilt explained how every pipeline run is a function of the code, environment, and data, and went on to show how Quilt could help dramatically simplify data management with dataset versioning, accessibility, and verifiabi ...
```
```suggestion
In his talk with the clever title “[one link to rule them all](https://www.youtube.com/watch?v=dttkcuP3OBc&list=PLPZ8WHdZGxmUdAJlHowo7zL2pN3x97d32&index=13),” Aneesh Karve of Quilt explained how every pipeline run is a function of the code, environment, and data, and went on to show how Quilt could help dramatically simplify data management with dataset versioning, accessibility, and verifiability. Data provenance and traceability were also front and center when Yih-Chii Hwang of DNAnexus described her team’s work around [bringing GxP compliance to Nextflow workflows](https://www.youtube.com/watch?v=RIwpJTDlLiE&list=PLPZ8WHdZGxmUdAJlHowo7zL2pN3x97d32&index=21).
```

abhi18av commented on content/blog/2022/nextflow-summit-2022-recap.md:
```diff
@@ -0,0 +1,92 @@
+
+## Compute optimization
+
+Another recurring theme was improving compute efficiency. Several talks discussed using containers more effectively, leveraging GPUs & FPGAs for added performance, improving instance type selection, and automating resource requirements. Mike Smoot of Illumina talked about Nextflow, Kubernetes, and DRAGENs and how Illumina’s FPGA-based Bio-IT Platform can dramatically accelerate analysis. Venkat M ...
```
```suggestion
Another recurring theme was improving compute efficiency. Several talks discussed using containers more effectively, leveraging GPUs & FPGAs for added performance, improving virtual machine instance type selection, and automating resource requirements. Mike Smoot of Illumina talked about Nextflow, Kubernetes, and DRAGENs and how Illumina’s FPGA-based Bio-IT Platform can dramatically accelerate analysis. Venkat Malladi discussed efforts to suggest optimal VM types based on different standardized nf-core labels in the Azure cloud (process_low, process_medium, process_high, etc.) Finally, Evan Floden discussed [Nextflow Tower](https://www.youtube.com/watch?v=yJpN3fRSClA&list=PLPZ8WHdZGxmUdAJlHowo7zL2pN3x97d32&index=22) and unveiled an exciting new [resource optimization feature](https://seqera.io/blog/optimizing-resource-usage-with-nextflow-tower/) that can intelligently tune pipeline resource requests to radically reduce cloud costs and improve run speed. Overall, the Nextflow community continues to make giant strides in improving efficiency and managing costs in the cloud.
```

abhi18av commented on content/blog/2022/nextflow-summit-2022-recap.md:
```diff
@@ -0,0 +1,92 @@
+
+## Beyond genomics
+
+While most summit speakers focused on genomics, a few discussed data pipelines in other areas, including statistical modeling, analysis, and machine learning. Nicola Visonà from Università degli Studi di Macerata gave a fascinating talk about [using agent-based models to simulate the first industrial revolution](https://www.youtube.com/watch?v=PlKJ0IDV_ds&list=PLPZ8WHdZGxmUdAJlHowo7zL2pN3x97d32&i ...
```
```suggestion
While most summit speakers focused on genomics, a few discussed data pipelines in other areas, including statistical modeling, analysis, and machine learning. Nicola Visonà from Università degli Studi di Macerata gave a fascinating talk about [using agent-based models to simulate the first industrial revolution](https://www.youtube.com/watch?v=PlKJ0IDV_ds&list=PLPZ8WHdZGxmUdAJlHowo7zL2pN3x97d32&index=27). Similarly, Konrad Rokicki from the Janelia Research Campus explained how Janelia are using [Nextflow for petascale bioimaging data](https://www.youtube.com/watch?v=ZjSzx1I76z0&list=PLPZ8WHdZGxmUdAJlHowo7zL2pN3x97d32&index=18) and why bioimage processing remains a large domain area with an unmet need for reproducible workflows.
```

abhi18av commented on content/blog/2022/nextflow-summit-2022-recap.md:
```diff
@@ -0,0 +1,92 @@
+
+## Summit Announcements
+
+This year’s summit also saw several exciting announcements from Nextflow developers. During his talk on [the future of containers](https://www.youtube.com/watch?v=PTbiCVq0-sE&list=PLPZ8WHdZGxmUdAJlHowo7zL2pN3x97d32&index=14), Paolo Di Tommaso announced the availability of [Nextflow 22.10.0](https://github.com/nextflow-io/nextflow/releases/tag/v22.10.0). In addition to various bug fixes, the lates ...
```
```suggestion
This year’s summit also saw several exciting announcements from Nextflow developers. Paolo Di Tommaso, during his talk on [the future of containers](https://www.youtube.com/watch?v=PTbiCVq0-sE&list=PLPZ8WHdZGxmUdAJlHowo7zL2pN3x97d32&index=14), announced the availability of [Nextflow 22.10.0](https://github.com/nextflow-io/nextflow/releases/tag/v22.10.0). In addition to various bug fixes, the latest Nextflow release introduces an exciting new technology called Wave that allows containers to be built on the fly from Dockerfiles or Conda recipes saved within a Nextflow pipeline. Wave also helps to simplify containerized pipeline deployment with features such as “container augmentation”; enabling developers to inject new container scripts and functionality on the fly without needing to rebuild the base containers such as a cloud-native [Fusion file system](https://www.nextflow.io/docs/latest/fusion.html). When used with Nextflow Tower, Wave also simplifies authentication to various public and private container registries. The latest Nextflow release also brings improved support for Kubernetes and enhancements to documentation, along with many other features.
```

abhi18av commented on content/blog/2022/nextflow-summit-2022-recap.md:
```diff
@@ -0,0 +1,92 @@
+
+Several other announcements were made during [Evan Floden’s talk](https://www.youtube.com/watch?v=yJpN3fRSClA&list=PLPZ8WHdZGxmUdAJlHowo7zL2pN3x97d32&index=22&t=127s), such as:
+
+- MultiQC is joining the Seqera Labs family
```
```suggestion
- MultiQC is joining the Seqera Labs family of products
```

abhi18av reviewed (approved):
Added minor comments, but otherwise LGTM.

pditommaso commented:
There are several comments unresolved. Noel as the author should take care of them

pditommaso commented:
I've merged this, but still think the pic can be improved. Maybe @liliagrc can work a bit on that?

nuelete commented:
I will ask her to work on the picture, but let's please publish it in the meantime, so that we can send out the newsletter tomorrow. Thanks :)

pditommaso commented:
merged == already online 😎

nuelete commented:
Lol, I know, but I didn’t see it online. Cleaning cache…


On Thu, 3 Nov 2022 at 19:41, Paolo Di Tommaso ***@***.***>
wrote:

> merged == already online 😎
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/nextflow-io/website/pull/73#issuecomment-1302525178>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A3WIOIXOQQ3QZHUWSN6M6A3WGQBOBANCNFSM6AAAAAARTDGFBQ>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>
-- 

Noel Ortiz

Head of Growth & Marketing

+34 623 04 10 94

***@***.***

Barcelona

<https://www.linkedin.com/in/noelortiz/> <https://twitter.com/nuelete>
<https://api.whatsapp.com/send?phone=34623041094>

<https://seqera.io/>

pditommaso commented:
Umm. actaully I cannot see it either

On Thu, Nov 3, 2022 at 7:45 PM nuelete ***@***.***> wrote:

> Lol, I know, but I didn’t see it online. Cleaning cache…
>
>

> Message ID: ***@***.***>
>

pditommaso commented:
Indeed, the frontmatter was messed up e8dd41c4d941. fixed
